.gitignore

```diff
@@ -1,2 +1,20 @@
-config.log
-config.status
+Makefile.in
+aclocal.m4
+ar-lib
+autom4te.cache/
+autoscan.log
+compile
+config.guess
+config.h.in
+config.sub
+configure
```

Comment:
We need configure.
Users must be able to build the project directly from the repository without having to install GNU auto tools.
That's how it used to work.